src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int mrn = 100;
+
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        String colorBall = color.getRandomColor();
+        Random random = new Random();
```

Comment:
let's make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method.